environments/staging/terraform.yml

```diff
@@ -422,3 +422,39 @@ efs_file_systems:
     create_record: yes
     domain_name: "staging.commcare.local"
     route_names: "shared-efs"
+
+existing_zone_record_update:
+    domain_name:  "staging.commcare.local"
+    records_list:
+      - route_names: rds_postgres
+        record_type: "CNAME"   ## Type CNAME supports a record only
+        records: 
+          - pgformplayer0-staging
+      - route_names: commcarehq_synclogs
+        record_type: "A"  ## Type A supports multiple records
+        records: 
+          - pgproxy2-staging
+      - route_names: proxy_form_processing
+        record_type: "A"  ## Type A supports multiple records
+        records: 
+          - pgproxy2-staging
+      - route_names: couchdb2_proxy_host
+        record_type: "A"    ## Type A supports multiple records
+        records: 
+          - couch6-staging
+      - route_names: formplayer_pgbouncer
+        record_type: "CNAME"    ## Type CNAME supports a record only
+        records: 
+          - pgformplayer_nlb-staging
```

Comment:
What if you try using this structure instead:

```
      - route_name: couchdb2_proxy_host
        record_type: "A"  ## Type A supports multiple records
        target_type: server
        targets: 
          - couch6-staging
      - route_name: formplayer_pgbouncer
        record_type: "CNAME"    ## Type CNAME supports a record only
        target_type: pgbouncer_nlb
        target: 
          - pgformplayer_nlb-staging
```

The differences are (1) `route_name` instead of `route_names` (2) `target` instead of `records` (since I think the whole thing is called a "record") and (3) adds a `target_type` field that right now is either `server` or `pgbouncer_nlb`. I'll leave a comment below about how `target_type` would be used.